custom_components/ui_lovelace_minimalist/lovelace/translations/default.yaml

```diff
@@ -4,6 +4,9 @@ ulm_translation_engine:
     ulm_translation_back: "[[[ return hass.resources[hass['language']]['ui.common.back']; ]]]"
     ulm_translation_brightness: "[[[ return hass.resources[hass['language']]['ui.card.light.brightness']; ]]]"
     ulm_translation_color_temperature: "[[[ return hass.resources[hass['language']]['ui.card.light.color_temperature']; ]]]"
+    ulm_translation_status: "[[[ return hass.resources[hass['language']]['ui.dialogs.more_info_control.vacuum.status']; ]]]"
```

Comment:
Might want to switch to a general entity once available, should be fine for now.